src/sentry/utils/committers.py

```diff
@@ -229,6 +231,17 @@ def get_event_file_committers(project, event, frame_limit=25):
         {match for match in commit_path_matches for match in commit_path_matches[match]}
     )
 
+    logger.info(
+        "relevant_commits",
+        extra={
+            "relevant_commits": relevant_commits,
+            "projectId": project.id,
```

Comment:
@NisanthanNanthakumar We generally use snake case on the backend so `project_id`

Reply:
@scefali makes sense. updated!